ddtrace/contrib/pytest/__init__.py

```diff
@@ -23,8 +23,8 @@
    The ddtrace plugin for pytest has the side effect of importing the ddtrace
    package and starting a global tracer.
 
-   If this is causing issues for your pytest runs where traced execution of
-   tests is not enabled, you can deactivate the plugin::
+   While you can avoid this by setting ``DD_TRACE_ENABLED=False``, if this is causing issues for your pytest
```

Comment:
```suggestion
   While you can avoid this by setting ``DD_TRACE_ENABLED=False``, if this is still causing issues for your pytest
```

Reply:
I think the still made the wording clearer for me in the other one so maybe want to add here. Without it, it kind of hurts to read

Reply:
> Do we need to worry about the start_span here? https://github.com/DataDog/dd-trace-py/blob/1.x/ddtrace/contrib/pytest_bdd/plugin.py#L64 I took a quick look and it didn't seem to care if the tracer was enabled or not.

As long as `tracer.enabled` is set to false, the tracer's span processors responsible for writing traces won't run (their `on_span_start()` and `on_span_finish()` won't run). So we shouldn't need to worry about this 😄